code/game/machinery/vending.dm

```diff
@@ -444,26 +445,17 @@
 		return 0
 	else
 		// Okay to move the money at this point
-		var/paid = customer_account.charge(currently_vending.price, GLOB.vendor_account,
+		customer_account.charge(currently_vending.price, GLOB.vendor_account,
 			"Purchase of [currently_vending.name]", name, GLOB.vendor_account.owner_name,
 			"Sale of [currently_vending.name]", customer_account.owner_name)
 
-		if(paid)
-			// Give the vendor the money. We use the account owner name, which means
-			// that purchases made with stolen/borrowed card will look like the card
-			// owner made them
-			credit_purchase(customer_account.owner_name)
-		return paid
+		return 1
```

Comment:
`TRUE` here. Boolean values should always be denoted as `TRUE/FALSE`. Old code still uses `1` or `0`. Which TRUE or FALSE map to btw.